tensorflow/core/api_def/python_api/api_def_TensorScatterMax.pbtxt

```diff
@@ -0,0 +1,11 @@
+op {
+  graph_op_name: "TensorScatterMax"
+  deprecation_message: "Use tensor_scatter_nd_max instead"
```

Comment:
Why are you creating it with a deprecated endpoint already? Just add the new endpoint we want to keep.